scheduler/cloudformation/cloudformation.go

```diff
@@ -316,44 +422,29 @@ func (s *Scheduler) updateStack(stack *cloudformation.Stack, input *cloudformati
 		}
 	}
 
-	// If there's currently an update happening, wait for it to
-	// complete.
-	if strings.Contains(status, "IN_PROGRESS") {
```

Comment:
We should probably keep this around to ensure that the stack is in an updateable state before we start. Without this, creating a new stack then immediately deploying would cause an error.